app/res/values/themes.xml

```diff
@@ -119,8 +120,18 @@
     <style name="TextAppearance.Custom.WindowTitle.Case" parent="TextAppearance.Custom.WindowTitle">

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/5a959df2d1681e0a640ebb72d4513eaecefa585e/app/res/values/themes.xml#L120"></span>The resource <code>R.style.TextAppearance_Custom_WindowTitle_Case</code> appears to be unused</td>
    </tr>
  </tbody>
</table>